testing/python/transform/test_tilelang_transform_layout_inference.py

```
@@ -100,6 +100,29 @@ def main(
+def test_copy_layout_uses_active_threads_for_ragged_thread_count():
+    block_M = 256
+    block_N = 128
+    threads = 384
+    dtype = T.bfloat16
+
+    @T.prim_func
+    def main(
+        Bias: T.Tensor((block_M, block_N), dtype),
+        C: T.Tensor((block_M, block_N), dtype),
+    ):
+        with T.Kernel(1, 1, threads=threads) as (bx, by):
```

Ruff (0.15.14)

[warning] Unpacked variable `bx` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)